terraform/app_clients/secrets.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  description = "Storage service credentials; managed in https://github.com/wellcomecollection/storage-service/tree/master/terraform/app_clients"
+}
+
+# Store the Catalogue client credentials in KMS for the METS adapter
```

Comment:
Should these go in the platform account so they can be extracted through SSM easily?

Reply:
1. They’re already in the platform account – it owns the Cognito User Pool, so that’s the Terraform provider I’m using.

2. The METS adapter already pulls from these SSM parameters, but now they’re managed automatically rather than somebody copying them across by hand.